osf/models/collection.py

```diff
@@ -39,6 +45,72 @@ class Meta:
     program_area = models.CharField(blank=True, max_length=127)
     school_type = models.CharField(blank=True, max_length=127)
     study_design = models.CharField(blank=True, max_length=127)
+    machine_state = models.IntegerField(
```

Comment:
Do we also want/need a `deleted` field? Or is a `REMOVED` state good enough for filtering?

Reply:
Was this ever raised in scrum or answered?